lib/auth/v2/checkRequestExpiry.ts

```diff
@@ -14,19 +15,19 @@ export default function checkRequestExpiry(timestamp: number, log: RequestLogger
     // timestamp is more than 15 minutes in the future, the request
     // has expired and return errors.RequestTimeTooSkewed
     const currentTime = Date.now();
-    log.trace('request timestamp', { requestTimestamp: timestamp });
-    log.trace('current timestamp', { currentTimestamp: currentTime });
-
-    const fifteenMinutes = (15 * 60 * 1000);
-    if (currentTime - timestamp > fifteenMinutes) {
-        log.trace('request timestamp is not within 15 minutes of current time');
-        log.debug('request time too skewed', { timestamp });
+    if (currentTime - timestamp > fifteenMinutesInMilliseconds) {
+        log.debug('request timestamp is not within 15 minutes of current time', {
```

Comment:
nit: keeping the "original" debug log maybe more friendly for support, if they are used to searching for `request time too skewed`

```suggestion
        log.debug('request time too skewed, timestamp not within 15 minutes of current time', {
```